grails-datastore-core/src/main/groovy/org/grails/datastore/mapping/model/types/Association.java

```diff
@@ -55,6 +62,25 @@ public Association(PersistentEntity owner, MappingContext context, String name,
         super(owner, context, name, type);
     }
 
+    private void buildCascadeOperations() {
+        String cascade = this.getMapping().getMappedForm().getCascade();
```

Comment:
The value of `cascade` can be a comma separated string containing multiple values